test/composertest.py

```diff
@@ -135,6 +136,19 @@ def setUp(self):
         r = subprocess.run(self.ssh_command + curl_command, stdout=subprocess.DEVNULL)
         self.assertEqual(r.returncode, 0)
 
+        # Make sure the right backend is running
+        print("Checking to make sure backend is %s" % os.getenv("BACKEND"))
+        curl_command = ["curl", "--max-time", "360",
+                                "--silent",
+                                "--unix-socket", "/run/weldr/api.socket",
+                                "http://localhost/api/status"]
+        r = subprocess.run(self.ssh_command + curl_command, stdout=subprocess.PIPE)
+        self.assertEqual(r.returncode, 0)
+        status = json.loads(r.stdout)
+        print(status)
+        self.assertTrue("backend" in status)
+        self.assertEqual(os.getenv("BACKEND"), status["backend"])
```

Comment:
@bcl I'm not sure I understand this block. It duplicates the curl_command and almost duplicates subprocess.run. Did you mean to execute this twice or simply forgot to delete the old block ?

Reply:
Yeah, didn't clean up things after I got it working. Except that in #1007 it looks like the return code is 20 for some reason, and curl doesn't document what 20 means...